scanner/matcher/matcher.go

```diff
@@ -160,3 +161,18 @@ func (m *matcherImpl) Close(ctx context.Context) error {
 	m.pool.Close()
 	return err
 }
+
+// Ready return true if the matcher is ready.
+func (m *matcherImpl) Ready(ctx context.Context) error {
+	ok, err := m.libVuln.Initialized(ctx)
+	if err != nil {
+		return err
+	}
+	if !ok {
+		return errors.New("matcher vulnerability store is not initialized")
+	}
+	if err := m.pool.Ping(ctx); err != nil {
```

Comment:
maybe check this first since this pool is what is used for `Initialized` anyway

Reply:
The first will skip the DB check [after the first confirmation](https://github.com/quay/claircore/blob/main/datastore/postgres/initialized.go#L8-L29). Keeping it first to me seems more idiomatic for that reason.

Reply:
I removed the previous check because the POD would never be ready in offline mode until the first vulns were loaded from Central. Resolving this.